views/video.handlebars

```diff
@@ -72,7 +72,6 @@
 
             <!-- <a href="/updateStance">Return to Profile</a> -->
         </div>
-<button id="end-conversation" class="btn btn-success" disabled="true">End Conversation</button>
 <br><br><br><br>
 </div>
 
```

Comment:
in line 133 change "Learn something from them." to "Try to learn something from them!"